Source/Core/VideoCommon/UberShaderPixel.cpp

```diff
@@ -367,21 +367,21 @@ ShaderCode GenPixelShader(APIType ApiType, const ShaderHostConfig& host_config,
       "// which are common to both color and alpha channels\n"
       "bool tevCompare(uint op, int3 color_A, int3 color_B) {{\n"
       "  switch (op) {{\n"
-      "  case 0u: // TEVCMP_R8_GT\n"
+      "  case 0u: // TevCompareMode::R8, TevComparison::GT\n"
```

Comment:
Surprised these aren't using the `case {:s}:\n` trick

Reply:
It's because there are two enums, instead of one pseudo-enum/set of random values that don't really make sense.  The values were previously these (note how they don't match what's in the switch statement):

```
TEVCMP_R8_GT = 8,
TEVCMP_R8_EQ = 9,
TEVCMP_GR16_GT = 10,
TEVCMP_GR16_EQ = 11,
TEVCMP_BGR24_GT = 12,
TEVCMP_BGR24_EQ = 13,
TEVCMP_RGB8_GT = 14,
TEVCMP_RGB8_EQ = 15,
TEVCMP_A8_GT = TEVCMP_RGB8_GT,
TEVCMP_A8_EQ = TEVCMP_RGB8_EQ
```

and the enum was used by `((cc.shift << 1) | cc.op | 8)`.  I decided to just split it up into two enums; for the software renderer I was able to split it into two parts (acting on `cc.compare` and then looking at whether it was using eq or gt), but for the hardware backends I left it in the combined form since it would require further restructuring for no real gain.

It does look like there are other switch statements in this file that hardcode enum values, though; those could be changed.

Reply:
Due to the binary search tree structure used for the other enums (in addition to the switch statements), I've decided not to deal with the hardcoded enum values in this PR; it can be handled in a later one.